### PR TITLE
Bugfix/fix before migration

### PR DESCRIPTION
**Pull request by Avareco**

### CODE REVIEW

File: src/components/AddItemDialog.vue [hunk 1]
```diff
@@ -98,68 +103,113 @@ function getRootItemPath(): string {
...
-function addIcon(e: FileUploadSelectEvent) {
+function addIcon(e: FileUploadSelectEvent): void {
   addForm.icon = e.files[0].objectURL;
 }
-function removeIcon() {
+function removeIcon(): void {
   addForm.icon = "";
 }
+
+onMounted(() => {
```
Comment (hollote) [inline src/components/AddItemDialog.vue:195]: replace with one watcher

**Review by hollote — CHANGES_REQUESTED** [review]
please fix leftovers

File: src/components/AddItemDialog.vue (hunk 1 above)
Comment (Avareco) [reply src/components/AddItemDialog.vue:201]: We don`t need onUpdated now. Only onMounted left

**Review by hollote — APPROVED** [review]

File: src/components/Filters.vue [hunk 2]
```diff
@@ -49,17 +49,18 @@ function onResetFilters(): void {
 }
 
 function setPrevFilterValue(): void {
-  if (
-    allTags &&
-    allTags
-      .filter((tag) => tag.favorite)
-      .find((tag) => tag.name === props.tagForSearch)
-  ) {
+  if (allTags && allTags) {
```
Comment (hollote) [inline src/components/Filters.vue:52]: ?

File: src/components/Filters.vue [hunk 3]
```diff
@@ -81,7 +82,7 @@ onMounted(() => {
   <h3>Filter by favorite tag:</h3>
   <Dropdown
     v-model="selectedFavTag"
-    :options="allTags && allTags.filter((tag) => tag.favorite)"
+    :options="allTags && favoriteTags"
```
Comment (hollote) [inline src/components/Filters.vue:85]: favoriteTags will be enough

**Review by hollote — CHANGES_REQUESTED** [review]
please fix comments

File: src/components/Filters.vue (hunk 2 above)
Comment (Avareco) [reply src/components/Filters.vue]: fixed

File: src/components/Filters.vue (hunk 3 above)
Comment (Avareco) [reply src/components/Filters.vue]: fixed

**Pull request merged**